mantra.md

```diff
@@ -22,10 +22,43 @@
 8. ACK Triangle
 ===============
 
+Contrary to belief of many new-age engineering shops; the utmost elegant
+products must be built in a environment constricted by bureaucratic hurdles.
+Not only does this does it prove that the product is fit enough for survival
+for years to come, but also serves as ego re-enforcement of the involved
+fiefdoms.
+
+While it is true that, the more fiefdoms a change must clear before it is
+declared viable, the better. There are diminishing returns. New-agers
```

Comment:
To make it read more naturally, I'd leave out the "While" from the first sentence and add a "However," for the second.